### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by progproc**
First draft of the Czech translation. Will amend if my proofreaders find anything. 
I checked this and it works on desktop and in a phone web browser.

Some labels were longer than in English but I think it's better to have it longer rather than simplify too much.

Notes:
The headline means **"The surprising efficacy of surgical face masks and respirators"**

I use the translation of "roušky a respirátory" (surgical face masks and respirators) or "ochrana obličeje" (face protection), since "face mask" does not have an easy translation that would make sense in the context of this essay.

### CODE REVIEW

**Comment by progproc** [comment]
I added some more corrections from my Twitter followers.

**Comment by aatishb** [comment]
Thanks! I'll work on bringing this in sync with the live branch in the next few days. Once I'm done will let you know if any steps remain.

File: index-cs.html [hunk 1]
```diff
@@ -0,0 +1,748 @@
...
+        <slider v-model="eout" :min="0" :max="0.99" :step="0.01"></slider>
+      </label>
+
+      <label class="row half bigsplash">
+        <div>Účinnost ochrany (nádech): <span class="dotted">{{convertToPercent(ein)}}</span>%</div>
+        <slider v-model="ein" :min="0" :max="0.99" :step="0.01"></slider>
+      </label>
+
+      <graph :traces="graph3Traces(
+        '% Infected'
```
Comment (aatishb) [inline index-cs.html:624]: @progproc we added a new line of code here so that this hover text can be translated. Could you please translate this phrase?

**Review by aatishb — CHANGES_REQUESTED** [review]
Hi @progproc. Thanks for all your work on this, it looks great. I brought this up to date with the live version. The only thing remaining is one small translation, you can do this by committing the change to your branch as before, and it should automatically appear here. Or just let me know the translation and I can swap it in. Thanks!

**Comment by aatishb** [comment]
Tagging #4

**Comment by progproc** [comment]
I am only starting with github so not sure what to do. I guess this would
be simpler:

Translation:
Infected = Infikovaných

Thanks

**Comment by progproc** [comment]
People are definitely liking the essay. Thanks again!

**Review by aatishb — APPROVED** [review]

**Comment by aatishb** [comment]
@progproc it's live! Thanks for all your efforts on this. Glad to see it reach a wider audience.

**Comment by aatishb** [comment]
@all-contributors add @progproc for translation

**Pull request merged**